### ISSUE TITLE
hideSplashscreen on capacitor causes missing onMounted

### ISSUE THREAD

**Issue by francoism90** [quasarframework/quasar#9079]
**Describe the bug**
```
$ quasar build --mode capacitor --target android --ide
```

Inspect info by Chromium:
```
native App.addListener (#59478500)
vendor.js:3 ReferenceError: onMounted is not defined
    at setup (app.js:1)
    at o (vendor.js:3)
    at En (vendor.js:3)
    at Tn (vendor.js:3)
    at D (vendor.js:3)
    at N (vendor.js:3)
    at y (vendor.js:3)
    at Q (vendor.js:3)
    at mount (vendor.js:3)
    at Object.t.mount (vendor.js:3)
```

**Codepen/jsFiddle/Codesandbox (required)**
This only happens in capacitor, not in SPA/PWA.

**To Reproduce**
```
capacitor: {
      hideSplashscreen: true
    },
```

I'm using TS, don't know if this is related?

**Expected behavior**
Import onMounted from Vue.

**Platform (please complete the following information):**
Quasar Version:  `v2.0.0-beta.13`
@quasar/app Version: `v3.0.0-beta.17`
Quasar mode:
  - [ ] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [X ] Capacitor
  - [ ] BEX

Tested on:
  - [ ] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [X ] Capacitor
  - [ ] BEX

OS: Arch Linux
Node: v16.0.0
NPM: 7.11.1
Yarn: N/A
Browsers: Firefox/Chrome/Webrender
iOS: N/A
Android: 10
Electron: N/A

**Additional context**
Workaround:
```
 // Full list of options: https://v2.quasar.dev/quasar-cli/developing-capacitor-apps/configuring-capacitor
    capacitor: {
      hideSplashscreen: false
    },
```
I tried to fix the import, but I don't know the best way to edit app.template so it includes the onMounted() method, sorry.

**Comment by rstoenescu** [comment]
Fix will be available in "@quasar/app" v3.0.0-beta.18

**Comment by francoism90** [comment]
@rstoenescu Thanks for the fast fix! Keep up the good work, really like Quasar V2 so far. :)
